internal/wallet/wallet.go

```diff
@@ -23,8 +23,10 @@ type Wallet interface {
 
 	NewPubkey() (*btcec.PublicKey, error)
 	NewWitnessPubkeyScript() (pkScript []byte, err error)
-	ListUnspent() (utxos []Utxo, err error)
+	ListUnspent() (utxos []*Utxo, err error)
 
+	Manager() *waddrmgr.Manager
+	TxStore() *wtxmgr.Store
```

Comment:
Added getter functions so functions that use the Wallet interface have access to wallet's manager and txStore